main/heart_stokes.cc

```diff
@@ -0,0 +1,205 @@
+#include "interfaces/stokes.h"
+#include "pidomus.h"
+
+#include "deal.II/base/numbers.h"
+
+#include "Teuchos_CommandLineProcessor.hpp"
+#include "Teuchos_GlobalMPISession.hpp"
+#include "Teuchos_oblackholestream.hpp"
+#include "Teuchos_StandardCatchMacros.hpp"
+#include "Teuchos_Version.hpp"
+
+#include "mpi.h"
+#include <iostream>
+#include <string>
+
+#include "elastic_problem.h"
+
+////////////////////////////////////////////////////////////////////////////////
+// Begin macros
+#define problem_Stokes(dim,spacedim,LAC) \
```

Comment:
you don't need to define a macro here, since you are only using one interface (stokes).